common/src/main/java/com/wynntils/models/character/CharacterModel.java

```diff
@@ -125,6 +126,11 @@ public boolean isVeteran() {
         return isVeteran;
     }
 
+    public boolean isHunted() {
+        return !LoreUtils.getStringLore(McUtils.inventory().getItem(SOUL_POINT_SLOT))
+                .contains(HUNTED_STRING);
```

Comment:
Let's not use contains. Is there a reason not to match the whole string, either with a pattern or a hardcoded string?